### ISSUE TITLE
Globally define __root etc. for imports

### ISSUE THREAD

**Issue by wpf500** [thebristolcable/membership-system#70]
Most JS files start with something like:

```
var __root = __dirname + '/../..',
	__src = __root + '/src',
	__js = __src + '/js',
	__models = __src + '/models';
```

This is a bit verbose, we could remove all of this in favour of one of the strategies suggested [here](https://gist.github.com/branneman/8048520). Preferred option is the 3. The Global as it doesn't use any extra modules or monkey patching

**Comment by wpf500** [comment]
Partly addressed by #76

**Comment by wpf500** [comment]
Switched to `module-alias` in #219
